guide/english/r/objects/index.md

```diff
@@ -36,19 +36,29 @@ The following creates the function circleArea that calculates the area of a circ
 1. To create an R object, choose a name and then use the less-than symbol, `<`,
 followed by a minus sign,  `-`, to save data into it. This combination looks like an
 arrow, `<-`. R will make an object, give it your name, and store in it whatever
-follows the arrow. You can also use the '=' sign, but this is a less common method to assign a value to an object and generally frowned upon.
+follows the arrow. You can also use the '=' sign, but this is a less common method to assign a value to an object and generally frowned upon. There is one function named `assign` in R, which can also be used as an assignment operator. 
```

Comment:
As a suggestion, adding a line that tells the reader that he/she will find an example listed below that will explain the use of all the above methods of object creation would be a good addition, as:
This will increase the overall readability of the whole article and will also link the paragraphs that you write and the corresponding code examples.

Reply:
Thank you for your suggestion. I have added example for `assign` operator in point number 2, where the other assignment examples had been added.